### DISCUSSION TITLE
Should you add auto-generated prisma & graphql schema files to git-ignore?

### DISCUSSION THREAD

**Discussion by NHDang95** [prisma/prisma#11426] [Q&A] [answered]
As the title suggests, our team is working on the same repo, and I don't know if adding those 2 files is going to affect any aspects of any of the team members' work.

**Comment by Skryvvara** [comment, marked as answer]
I don't know about the graphql schema but if you not include the prisma schema your team members cant run prisma generate thus typescript wont be able to import the types

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!

**Comment by spi9325** [comment]
prisma schema is important.if you start deploying and not found these flie then also error comes.
